Carrot-Assistant/requirements.txt

```diff
@@ -11,4 +11,4 @@ streamlit==1.36.0
 rapidfuzz==3.5.2
 sqlalchemy==2.0.30
 openssl=3.1.4
```

Comment:
We had a discussion to remove openssl and it was removed earlier yeah?

Reply:
Just testing 😛